drivers/SmartThings/zigbee-contact/src/test/test_aqara_contact_sensor.lua

```diff
@@ -86,32 +86,34 @@ test.register_coroutine_test(
   end
 )
 
-test.register_coroutine_test(
-  "closed contact events",
-  function()
-    local attr_report_data = {
-      { IASZone.ID, data_types.Int16.ID, 0x0020}
-    }
-    test.socket.zigbee:__queue_receive({
-      mock_device.id,
-      zigbee_test_utils.build_attribute_report(mock_device, IASZone.ID, attr_report_data, 0x115F)
-    })
-    test.socket.capability:__expect_send( mock_device:generate_test_message("main", capabilities.contactSensor.contact.closed()))
-  end
-)
+--TODO There are no handlers defined for these attr reports
+-- This and other subdrivers are not handling these reports
+-- test.register_coroutine_test(
+--   "closed contact events",
+--   function()
+--     local attr_report_data = {
+--       { IASZone.ID, data_types.Int16.ID, 0x0020}
```

Comment:
This line is wrong, it needs to be:

```
{ IASZone.attributes.ZoneStatus.ID, data_types.Bitmap16.ID, 0x0020}
```
and the same is true in the other test.

Reply:
> The subdriver is saying it can handle the aqara IASZone updates, but the subdriver is not specifying handlers for that attribute, so they are just dropped because no handler is found. When I run tests locally I can confirm that the defaults are not being called either. When I add tests for other subdrivers I see them fail as well; only those that use the base driver impl pass the contact sensor tests.

^^ this is not true, which is what I was almost sure of, but I just couldn't figure out why the defaults weren't called. Ill make the test changes then.